example/demo/tsconfig.json

```diff
@@ -5,10 +5,14 @@
 		"lib": ["es2021"],
 		"module": "es2022",
 		"moduleResolution": "Bundler",
-		"types": ["@cloudflare/workers-types"]
+		"types": ["@cloudflare/workers-types"],
+		"jsx": "react-jsx",
+    "jsxImportSource": "@final-ui/react"
```

Comment:
All that is needed to enable custom JSX. We would ship a preset so this does not need to be set by users.

Reply:
Ok I guess this means that users cannot use JSX if their server code also uses React SSR.

Shouldn't be a big deal but it does seem unfortunate

Reply:
Yeah, or... we could use Babel transformer with conditionals (or custom file extension). I guess we can provide this in "advanced" section of some guides we make.